xbmc/platform/darwin/ios-common/PlatformDarwinEmbedded.cpp

```diff
@@ -53,3 +53,8 @@ bool CPlatformDarwinEmbedded::InitStageTwo()
 
   return true;
 }
+
+bool CPlatformDarwinEmbedded::SupportsUserInstalledBinaryAddons()
+{
+  return false;
+}
```

Comment:
maybe this and the one for `Win10` inline into the header?
 `bool SupportsUserInstalledBinaryAddons() { return false; }`
you could see immediately what's going on and save lots of disc space (;;;

Reply:
Well no. At the end its persoanl taste, and as we have no hard rules for this, I'd like to divide declaration and implementation. Maybe not of real concern for this case, but generally I prefer only to recompile one cpp instead of all cpps which include the changed header.

Reply:
> you could see immediately what's going on

Yeah, let's put all our code in one header file. ;-)

Reply:
>  I prefer only to recompile one cpp instead

you win :)